src/models/UserModel.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose'
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
import mongoose from 'mongoose';
```